src/app/shared/http-services/current-user.service.ts

```diff
@@ -34,12 +50,11 @@ export class CurrentUserHttpService {
       .get<RawUserProfile>(`${appConfig().apiUrl}/current-user`)
       .pipe(
         map(raw => ({
+          ...raw,
```

Comment:
Check get-item-by-id and get-group-by-id. The get should be to an `unknown` type and then just decode.
Stick to the attribute names used in the backend, so use `groupId` and `tempUser` everywhere (may require to change other places of the code).

Reply:
Fixed